README.md

```diff
@@ -1,2 +1,2 @@
 # durer_login
-Log in page for Durer competition.
+Login page for Durer competition.
```

Comment:
Do you sure, its written together?

Reply:
Yes, of course. Just look at it, it's obvious.

Reply:
Look now, it's even better.

Reply:
I see, see.`int main(){somecode}`